src/player/player.rs

```diff
@@ -278,7 +311,8 @@ impl SpotifyPlayer {
     }
 }
 
-const CLIENT_ID: &str = "782ae96ea60f4cdf986a766049607005";
+const REDIRECT_URI: &str = "http://127.0.0.1:8898/login";
+const SPOTIFY_CLIENT_ID: &str = "65b708073fc0480ea92a077233ca87bd";
```

Comment:
does the client id need to change? i guess we should eventually make this configurable

Reply:
updated my client to allow the expected redirect uris, if you want to revert that!